sidecars/ruby/PLATFORMS

```diff
@@ -0,0 +1 @@
+linux/amd64,linux/ppc64le,linux/s390x,linux/arm64
```

Comment:
#5 [linux/ppc64le internal] load metadata for docker.io/library/ruby:2.6.6-...

#5 ERROR: no match for platform in manifest sha256:c1187bcdc4a2f3862f78b329df367f640fe55e99d7c12385674b5fd3ae1d30e4: not found

#6 [linux/s390x internal] load metadata for docker.io/library/ruby:2.6.6-st...

#6 ERROR: no match for platform in manifest sha256:c1187bcdc4a2f3862f78b329df367f640fe55e99d7c12385674b5fd3ae1d30e4: not found

you may need to remove those from PLATFORMS file 
linux/ppc64le,linux/s390x